blocked-edges/4.10.38-node-creation-fails.yaml

```diff
@@ -0,0 +1,11 @@
+to: 4.10.38
+from: .*
+url: https://issues.redhat.com/browse/OCPBUGS-2628
+name: NodeCreationFails
+message: |-
+  Ading new nodes to the exsiting cluster will fail.
```

Comment:
We should mention something about OpenStack here and in the `name`.  Maybe `OpenStackNodeCreationFails` and:

> OpenStack clusters on 4.10.38 may fail to provision new nodes.

Or maybe it's machines?  Or something...